WordPress/WordPressDebug.entitlements

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>aps-environment</key>
+	<string>development</string>
+	<key>com.apple.developer.applesignin</key>
+	<array>
+		<string>Default</string>
+	</array>
+	<key>com.apple.developer.associated-domains</key>
+	<array>
+		<string>webcredentials:wordpress.com</string>
+		<string>applinks:wordpress.com</string>
+		<string>applinks:apps.wordpress.com</string>
+		<string>applinks:public-api.wordpress.com</string>
+		<string>applinks:links.wp.a8cmail.com</string>
+		<string>applinks:*.wordpress.com</string>
```

Comment:
👋 @momo-ozawa, is this intentional? We've removed it for the Jetpack target's entitlement files above to allow passkeys to work, so I'm not sure if this was added here on purpose or not.